### PR TITLE
Create 2018-06-02-freenode-jobs.md

### PR DESCRIPTION
**Pull request by christeld**
jobs.freenode.net announcement text

### CODE REVIEW

File: content/news/2018-06-02-freenode-jobs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author: christel
+date: 2018-06-02 18:28:00+00:00
+slug: jobs-freenode
+title: Announcement: jobs.freenode.net - New Service
+category: general
+category: freenode
+category: community
+category: announcements
+---
+Over the last two decades, we have found that a variety of freenode community members have reached out to us when they have been involved in the hiring process at their places of work. We have always been keen to support and promote relevant topics within the wider freenode communities, and we are excited to be soft-launching the new [jobs.freenode.net](https://jobs.freenode.net) website.
```
Comment (nhandler) [inline content/news/2018-06-02-freenode-jobs.md]: Clarify what "soft launch" means in this context.

File: content/news/2018-06-02-freenode-jobs.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+category: freenode
+category: community
+category: announcements
+---
+Over the last two decades, we have found that a variety of freenode community members have reached out to us when they have been involved in the hiring process at their places of work. We have always been keen to support and promote relevant topics within the wider freenode communities, and we are excited to be soft-launching the new [jobs.freenode.net](https://jobs.freenode.net) website. 
+
+We hope that the new site will provide a useful addition to the existing freenode projects, such as the IRC network and the [#live](https://freenode.live) conference. 
+
+# Hiring?
+Head over to [jobs.freenode.net](https://jobs.freenode.net) and add your job openings! The service is free to use, although we would be grateful for a donation towards the operating costs of freenode services and the #live conference in the event that you successfully match via the website.
```
Comment (nhandler) [inline content/news/2018-06-02-freenode-jobs.md]: If you are going to talk about donations, we should update the site to have a donations page again. Right now, there is no clear/obvious way to donate other than by sponsoring a server.

File: content/news/2018-06-02-freenode-jobs.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# FOSS or other peer-directed project on the hunt for volunteers?
+Why not add a post on the [jobs.freenode.net](https://jobs.freenode.net) site to see if you may be able to attract some contributors from the wider freenode and FOSS communities?
+
+# Looking for a new job or a volunteer role? 
+Keep an eye on [jobs.freenode.net](https://jobs.freenode.net) to see if something of interest is added. We will utilise wallops on the freenode IRC network to provide a brief summary of available roles once a week, for those wishing to receive these, please set yourself +W (/umode +W or /mode yournick +W).
```
Comment (nhandler) [inline content/news/2018-06-02-freenode-jobs.md]: Maybe replace "once a week" with "periodically" as I imagine given our track record, that is what will end up happening.

File: content/news/2018-06-02-freenode-jobs.md (hunk 1 above)
Comment (nhandler) [inline content/news/2018-06-02-freenode-jobs.md]: You mention this below, but call out at the top that this isn't just for paid jobs, but can be a means for FOSS projects to recruit volunteers.

File: content/news/2018-06-02-freenode-jobs.md (hunk 3 above)
Comment (ilbelkyr) [inline content/news/2018-06-02-freenode-jobs.md]: Also it's `+w`, not `+W` (capitalization) :)

File: content/news/2018-06-02-freenode-jobs.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+# FOSS or other peer-directed project on the hunt for volunteers?
+Why not add a post on the [jobs.freenode.net](https://jobs.freenode.net) site to see if you may be able to attract some contributors from the wider freenode and FOSS communities?
+
+# Looking for a new job or a volunteer role? 
+Keep an eye on [jobs.freenode.net](https://jobs.freenode.net) to see if something of interest is added. We will utilise wallops on the freenode IRC network to provide a brief summary of available roles periodically, for those wishing to receive these, please set yourself +w (/umode +w or /mode yournick +w). 
+
+# Feature requests, suggestions and feedback?
+The github repository can be found [here](https://github.com/freenode/freenodejobs), you can also drop us an e-mail to jobs@freenode.net or find us in #freenode-jobs on the freenode network.
```
Comment (emersonveenstra) [inline content/news/2018-06-02-freenode-jobs.md]: It might be a good idea to link to the webchat for #freenode-jobs

**Pull request merged**